test/type.query.test.js

```diff
@@ -277,4 +277,22 @@ describe('type: query', () => {
             return;
         });
     });
+    describe('Execute ================', () => {
+        it('Should start executing queries', async () => {
+            const execute = await handler.execute('testInstance/testBU', ['query']);
```

Comment:
you seem to have an async issue because both tests failed due to missing api-response files:

- [ ] add response files: `test\resources\9999999\automation\v1\queries\<KEY>\actions\start\post-response.json`

![image](https://github.com/Accenture/sfmc-devtools/assets/1917227/272d12d0-5b79-446c-8d68-32f33aed43e3)


please make sure you always run the test cases yourself locally and actually check the output of the cases that you add / update or that might be affected by your change in any way to ensure you don't miss things like this async issue. Here, the test case finishes before the mock-API call is executed and hence mocha does not pick up on the issue.

you can do so by running `npm run test`, or by hitting CTRL+SHIFT+T (might ask you to pick default test task once) or, if you only want to run one test class, by directly naming it like this (with the example of query test class):

`npx mocha test/type.query.test.js`

Reply:
another way of running tests, though the test log is then in the OUTPUT tab, not in the TERMINAL tab - which doesn't have color coding - is to use the VSCode extension [Mocha Test Explorer](https://marketplace.visualstudio.com/items?itemName=hbenl.vscode-mocha-test-adapter). I use that regularly as well

![image](https://github.com/Accenture/sfmc-devtools/assets/1917227/c1a10d96-baf7-4636-b1dc-a5d7e73d49e3)